modules/ROOT/pages/auradb/change-data-capture/overview.adoc

```diff
@@ -0,0 +1,44 @@
+[[change-data-capture-overview]]
+= Change Data Capture (CDC)
+:description: This section covers changing transaction log enrichment mode using the Aura Console.
+
+label:AuraDB-Professional[] 
+label:AuraDB-Enterprise[]
+
+Change Data Capture tracks changes to your data by querying the transaction logs for Creates, Updates and Deletes, so that you can update other applications or data sources.
+
+For more information see link:{neo4j-docs-base-uri}/cdc/current[Neo4j DBMS -> Change Data Capture (CDC)].
+
+Log enrichment mode can be set to `OFF`, `DIFF` or `FULL`. 
+For more information see link:{neo4j-docs-base-uri}/cdc/current/getting-started/enrichment-mode/[Neo4j DBMS -> CDC -> Enrichment mode].
+
+== Modifying log enrichment mode from within the Aura Console
+
+View an instance’s current log enrichment mode by clicking on an instance in the Aura Console. 
+
+Note that default log enrichment mode is `OFF` and change data capture is disabled until it is set as `DIFF` or `FULL`.
+
+Only Admins can change the log enrichment mode. 
+If an Admin changes log enrichment mode from `DIFF` or `FULL` to `OFF` applications dependent on CDC will stop working. For more information see link:https://neo4j.com/docs/cdc/current/getting-started/enrichment-mode/#_disabling_log_enrichment[Neo4j DBMS -> CDC -> Disabling Log Enrichment].
+
+== Performing a restore in Aura with log enrichment mode enabled
+
+If you take a snapshot and restore it to the same instance, or create a new instance from that snapshot, by default log enrichment mode will be set to `OFF`. 
+If the original status of that instance had log enrichment mode set to `DIFF` or `FULL`, the status changes. 
+You need to proactively enable it.
```

Comment:
```suggestion
To record changes when you take a snapshot and restore it to the same instance, you need to proactively set the log enrichment mode to `DIFF` or `FULL` since it is set to `OFF` by default.
```
I'm a little unsure what you are trying to say here, but this is an attempt to make it a little clearer.

Reply:
@l-heemann could you please take a few screenshots of this flow for us to see how it works please?

Reply:
I think screenshots need to wait until the button becomes available in Aura. Maybe somebody from the aura-console team who is working on this feature has access to a dev environment or mockup of what it will look like before we release it to customers. I doubt I'll have access to an aura environment with this button before it exists for customers.

For now I think we should maybe rephrase it to be more of a link to the CDC docs on this.